tests/zfs-tests/tests/functional/cli_root/zfs_rollback/zfs_rollback_common.kshlib

```diff
@@ -127,7 +126,15 @@ function setup_snap_env
 						log_must lockfs -f $TESTDIR1
 					fi
 				fi
+				if is_freebsd && [[ $dtst == $VOL ]]; then
```

Comment:
Can you add a comment here explaining why this is needed on FreeBSD

Reply:
:+1: done